src/bfx_tools/optitype.ml

```diff
@@ -4,8 +4,25 @@ open Common
 
 
 type product = <
-    is_done : Ketrew_pure.Target.Condition.t option ;
-    path: string >
+  host: Ketrew_pure.Host.t;
+  is_done : Ketrew_pure.Target.Condition.t option ;
+  path: string >
+
+let transform_optitype_product ?host ~path o =
```

Comment:
since the word “transform” is very vague it would be worth documenting those functions.
(and maybe `move_*_product` or `displace_*_product`?)